operator/src/test/java/org/keycloak/operator/KeycloakDeploymentE2EIT.java

```diff
@@ -246,4 +250,46 @@ public void testHostnameStrictDisabled() {
         }
     }
 
+    // Reference curl command:
+    // curl --insecure --data "grant_type=password&client_id=admin-cli&username=admin&password=adminPassword" https://localhost:8443/realms/master/protocol/openid-connect/token
+    @Test
+    public void testInitialAdminUser() {
+        try {
+            // Recreating the database to keep this test isolated
+            k8sclient.apps().statefulSets().inNamespace(namespace).withName("postgresql-db").delete();
```

Comment:
We might want to move this to `ClusterOperatorTest` as it comes in handy in other scenarios too.

Reply:
done